src/Symfony/Component/Translation/Tests/TranslatorCacheTest.php

```diff
@@ -213,54 +213,6 @@ public function testDifferentCacheFilesAreUsedForDifferentSetsOfFallbackLocales(
         $this->assertEquals('bar', $translator->trans('bar'));
     }
 
-    public function testPrimaryAndFallbackCataloguesContainTheSameMessagesRegardlessOfCaching()
```

Comment:
it shoud be removed in `3.0` rename it into `testLegacyPrimaryA..` instead and add 
```php
$this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
```